module/VuFind/src/VuFind/Recommend/PubDateVisAjax.php

```diff
@@ -135,7 +135,7 @@ public function process($results)
     public function getVisFacets()
     {
         // Don't bother processing if the result set is empty:
-        if ($this->searchObject->getResultTotal() == 0) {
+        if ($this->searchObject->getResultTotal() <= 0) {
```

Comment:
If you're trying to catch all the comparisons between getResultTotal() and 0, it looks like one got missed in \VuFindApi\Formatter\FacetFormatter.

Reply:
Fixed